### ISSUE TITLE
Load ephemerides for the asteroids to show their actual position on the map of Solar System

### ISSUE THREAD

**Issue by DarkStar1982** [DarkStar1982/Orbidium#4]
TL;DR: SSDB API offers "time since periapsis" value, so we can work from there.
Use this for reference: https://ssd.jpl.nasa.gov/sb/orbits.html

[Claude AI output - feels correct] 
To convert "time since periapsis" to a position angle on the elliptical orbit of a celestial body, we need to use Kepler's equations and the given orbital parameters. Here's a step-by-step process to achieve this:

Calculate the Mean Anomaly (M):
M = n * (t - T)
Where:
n = mean motion (2π / orbital period)
t = time since periapsis
T = time of periapsis passage (usually 0 if t is already "time since periapsis")
Solve Kepler's Equation to find the Eccentric Anomaly (E):
M = E - e * sin(E)
Where:
e = eccentricity of the orbit
This equation needs to be solved iteratively, as it's transcendental. You can use numerical methods like Newton-Raphson.
Calculate the True Anomaly (ν):
ν = 2 * arctan(sqrt((1 + e) / (1 - e)) * tan(E / 2))

The True Anomaly (ν) is the position angle you're looking for, measured from periapsis in the direction of motion.
Here's a Python function that implements this calculation

Code in Python for reference:
def time_to_true_anomaly(t, a, e, period):
    """
    Convert time since periapsis to true anomaly.
    
    Parameters:
    t : float
        Time since periapsis
    a : float
        Semi-major axis
    e : float
        Eccentricity
    period : float
        Orbital period
    
    Returns:
    float: True anomaly in radians
    """
    # Calculate mean motion
    n = 2 * np.pi / period
    
    # Calculate mean anomaly
    M = n * t
    
    # Solve Kepler's equation for eccentric anomaly
    E = M
    for i in range(100):  # Max 100 iterations
        E_next = E - (E - e * np.sin(E) - M) / (1 - e * np.cos(E))
        if abs(E_next - E) < 1e-8:  # Convergence check
            E = E_next
            break
        E = E_next
    
    # Calculate true anomaly
    nu = 2 * np.arctan(np.sqrt((1 + e) / (1 - e)) * np.tan(E / 2))
    
    return nu

# Example usage
t = 1000  # time since periapsis (in seconds)
a = 10000  # semi-major axis (in km)
e = 0.1  # eccentricity
period = 20000  # orbital period (in seconds)

true_anomaly = time_to_true_anomaly(t, a, e, period)
print(f"True anomaly: {np.degrees(true_anomaly)} degrees")

**Comment by DarkStar1982** [comment]
Closing - can do it without SSDB API call, using only MPC data field.
